examples/s2i-nginx/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# nginx-centos7
+# Here you can use whatever image base is relevant for your application.
+FROM centos:centos7
+
+# Here you can specify the maintaner for the image that you're building
+MAINTAINER Victor Palade <ipalade@redhat.com>
+
+# Export the environment variable that provides information about the application.
+# Replace this with the according version for your application.
+ENV NGINX_VERSION=1.6.3
+
+# Set the labels that are used for Openshift to describe the builder image.
+LABEL io.k8s.description="Nginx Webserver" \
+    io.k8s.display-name="Nginx 1.6.3" \
+    io.openshift.expose-services="8080:http" \
+    io.openshift.tags="builder,webserver,html,nginx" \
+    io.openshift.s2i.scripts-url="image:///usr/libexec/s2i"
+
+# Install our Nginx package and clean the yum cache so that we don't have any cached files waste space.
+RUN yum install -y epel-release  && \
+    yum install -y --setopt=tsflags=nodocs nginx && \
+    yum clean all
+
+# We will change the default port for nginx (This step is not required, but it's good to run services in containers on non-standard ports)
```

Comment:
maybe worth explaining that when you set this to "80" you won't be able to run this image other than "root"